### ISSUE TITLE
Bug - handling '|' char in filename

### ISSUE THREAD

**Issue by mthlvt** [DominikPieper/obsidian-ReadItLater#46]
Description: | in a title is not properly replaced and throws an invalid character error during note filename creation. 

Console output: 
```js
Uncaught (in promise) Error: File name cannot contain any of the following characters: * " \ / < > : | ?
    at we (app.js:1:244125)
    at t.checkPath (app.js:1:970824)
    at t.<anonymous> (app.js:1:972709)
    at app.js:1:235078
    at Object.next (app.js:1:235183)
    at app.js:1:234120
    at new Promise (<anonymous>)
    at m (app.js:1:233865)
    at t.create (app.js:1:972597)
    at ReadItLaterPlugin.eval (plugin:obsidian-read-it-later:6619:54)
```

How to reproduce: try to capture a youtube video containing a | char in the video title (example: https://www.youtube.com/watch?v=kPPq3dCe0Z0)

Tested on: plugin v 0.0.15 / Obsidian 0.15.9 on w11

**Comment by lukacovicadam** [comment]
Character "|" is striped from filename successfully. Problem is in the second double quote. Right now, normalizeFilename function stripes just first occurrence of character from list of illegal symbols. I'm working on fix.

**Comment by lukacovicadam** [comment]
Fixed in 0.0.16
